head/firmware/main.cpp

```diff
@@ -152,6 +153,18 @@ extern "C" void motor_callback_glue() {
     motor_interrupt_left.run_interrupt();
     motor_interrupt_right.run_interrupt();
 }
+adc::ADC_interface ADC_intf2 = {
+
+    .ADC_handle = &adc2};
+
+adc::ADC_interface ADC_intf1 = {
+
+    .ADC_handle = &adc1
+
+};
+
+adc::ADC ADC_comms(ADC_intf1, ADC_intf2);
+static presence_sensing_driver::PresenceSensingDriver psd(ADC_comms);
```

Comment:
nit: you use this style of initialization (`typename varname(ctor_args);`) here but the other style `auto varname = typename(ctor_args);` in `simulator/main.cpp`. can we use the same one (and ideally `auto`, but that's personal pref)